src/_sass/_colors.scss.template

```diff
@@ -39,7 +40,7 @@ $inverse_fill_color:                  rgba(%SEL_FG%, 0.04);
 $bg_color:             %BG%;
 $base_color:           %TXT_BG%;
 $alt_base_color:       %INACTIVE_TXT_BG%;
-$surface_color:        %TXT_BG%;
+$surface_color:        %BTN_BG%;
```

Comment:
To be honest, I'm not sure if this is OK, because `$surface_color` is used not only for buttons, but also for menus, popovers, popup dialogs, etc. (Just FYI, I'm also considering consolidating `$base_color` and `$surface_color` eventually as per Material Design guidelines.)

What do you think about this? If you are OK, I would not oppose this change.